### PR TITLE
Fix fte calculation and modified UI

### PR DESCRIPTION
**Pull request by GauravGusain98**
### Changes Include
- Fetch effortsheet data 2 times in a day #1467
- Remove monthly estimated hours input field for now
- Calculating FTE based on daily effort of users
- UI modifications

### CODE REVIEW

File: Modules/Project/Entities/Project.php [hunk 1]
```diff
@@ -91,4 +84,39 @@ public function getWorkingDaysList($startDate, $endDate)
...
     }
+
+    public function getCurrentExpectedHoursAttribute()
+    {
+        $teamMembers = $this->getTeamMembers()->get();
+        $updateDateCountAfterTime = config('efforttracking.update_date_count_after_time');
+        $currentDate = Carbon::now(config('constants.timezone.indian'));
+
+        if (Carbon::now(config('constants.timezone.indian'))->format('H:i:s') < $updateDateCountAfterTime) {
+            $currentDate = Carbon::now(config('constants.timezone.indian'))->subDay();
```
Comment (P4NK4J) [inline Modules/Project/Entities/Project.php:95]: Let's use helpers here

File: Modules/Project/Entities/ProjectTeamMember.php [hunk 2]
```diff
@@ -32,4 +32,22 @@ public function scopeActive($query)
...
+
+    public function getCurrentActualEffortAttribute()
+    {
+        return $this->projectTeamMemberEffort()->where('added_on', '>=', now(config('constants.timezone.indian'))->startOfMonth())->sum('actual_effort');
+    }
+
+    public function getCurrentExpectedEffortAttribute()
+    {
+        $project = new Project;
+        $daysTillToday = count($project->getWorkingDaysList(now(config('constants.timezone.indian'))->startOfMonth(), today(config('constants.timezone.indian'))));
```
Comment (P4NK4J) [inline Modules/Project/Entities/ProjectTeamMember.php:44]: Can we use the Project class method directly here?

File: Modules/Project/Resources/views/show.blade.php [hunk 3]
```diff
@@ -52,44 +46,95 @@
...
                     </div>
+                    <div class="form-group offset-md-1 col-md-5 pl-4">
+                        <h4 class="d-inline-block">
+                            <label for="name" class="font-weight-bold">Effortsheet:</label>
+                        </h4>
+                        @if($project->effort_sheet_url)
+                        <a id="view_effort_sheet_badge" href="{{ $project->effort_sheet_url }}" class="btn btn-primary btn-sm text-white ml-2 text-light rounded" target="_blank">{{ _('Open Sheet') }}</a>
+                        @else
+                        <span class="ml-2 fz-lg-22">Not Available</span>
+                        @endif
```
Comment (P4NK4J) [inline Modules/Project/Resources/views/show.blade.php:57]:
```suggestion
                        @if($project->effort_sheet_url)
                              <a id="view_effort_sheet_badge" href="{{ $project->effort_sheet_url }}" class="btn btn-primary btn-sm text-white ml-2 text-light rounded" target="_blank">{{ _('Open Sheet') }}</a>
                        @else
                             <span class="ml-2 fz-lg-22">Not Available</span>
                        @endif
```

**Review by P4NK4J — COMMENTED** [review]
Minor suggestions, else looks good.

File: Modules/Project/Entities/ProjectTeamMember.php (hunk 2 above)
Comment (GauravGusain98) [reply Modules/Project/Entities/ProjectTeamMember.php:44]: No it is showing warning of non static function being called statically.

File: Modules/Project/Entities/Project.php (hunk 1 above)
Comment (GauravGusain98) [reply Modules/Project/Entities/Project.php:95]: done

File: Modules/Project/Entities/ProjectTeamMember.php (hunk 2 above)
Comment (GauravGusain98) [reply Modules/Project/Entities/ProjectTeamMember.php:44]: deone

**Review by P4NK4J — APPROVED** [review]
👍

**Pull request merged**